cfme/tests/configure/test_paginator.py

```diff
@@ -6,6 +6,7 @@
 from cfme import test_requirements
 from cfme.configure.configuration.region_settings import RedHatUpdates
 from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.blockers import BZ
 
 general_list_pages = [
     ('servers', None, 'Details', False),
```

Comment:
_Optional_ consider using a `namedtuple` to help in explaining what each of these items in the tuple are for